rpc/ethereum/backend/feebackend.go

```diff
@@ -90,29 +91,28 @@ func (e *EVMBackend) processBlock(
 			if reward == nil {
 				reward = big.NewInt(0)
 			}
-			sorter[i] = txGasAndReward{gasUsed: txGasUsed, reward: reward}
-			break
```

Comment:
Why the `break`? I see this same question was asked by @fedekunze :

> I don't understand the purpose of this break here, are we only supporting 1 `MsgEthereumTx` message?
https://github.com/tharsis/ethermint/pull/734#discussion_r751158468

But I did not find the answer to it.

From what I understand, we should calculate the reward only from `MsgEthereumTx` and not from tendermint`Tx`. But I have some questions:
- can we find multiple `MsgEthereumTx` in a tendermint `Tx`? (`tx.GetMsgs()`)
- if yes, on what rules are the `MsgEthereumTx` batched in a `Tx`?